homework-submission/week1-project/src/Components/todoComponents.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+import React, { Component } from 'react';
+
+class Description extends Component{
+    render(){
+        return(
+            <h2>
+                * {this.props.descrip}, <Date deadline = {this.props.deadline}/> 
+            </h2>
+        );
+    }
+}
+
+class Date extends Component{
+    render(){
+        return(
+            <time dateTime={this.props.time}> {this.props.deadline}</time>
+        );
+    }
+}
+
```

Comment:
it is better to create every component in its own JS file, by this way you can reuse it everywhere